site/js/i18n-resources.js

```diff
@@ -1274,6 +1284,149 @@ var resources = { // English is fallback language.
+                "map is showing": 'Bản đồ hiển thị các đối tượng có thẻ <a rel="external" href="{{wikiUrl}}">opening_hours</a> như điểm tròn tô màu:',
+                "error": "Không thể phân tích giá trị",
+                "warning": "Các cảnh báo khi phân tích sẽ xuất hiện như {{sign}} màu xanh trong hình tượng trạng thái.",
+                "map filter": "Có thể áp dụng một số bộ lọc để phát hiện và khắc phục các lỗi (BĐCL) hoặc chỉ hiển thị các nơi đang mở cửa hoặc đóng cửa:",
+                "data source": 'Dữ liệu lớp phủ bắt nguồn từ {{OSMStartaTag}} và được truy vấn dùng {{APIaTag}}. Bản đồ là {{OSMaTag}}.',
+                "mode 0": 'Chỉ cho phép thời khoảng (các thẻ opening_hours, lit)',
+                "mode 1": 'Chỉ cho phép thời điểm',
+                "mode 2": 'Cho phép cả thời khoảng lẫn thời điểm (các thẻ service_times, collection_times)',
+                "value to long for osm": 'Giá trị quá dài đối với OSM. Cơ sở dữ liệu OpenStreetMap chỉ hỗ trợ các giá trị dài cho tới {{maxLength}} ký tự. Giá trị làm đẹp có {{pretLength}} ký tự, trong khi giá trị được nhập có {{valLength}} ký tự',
+                "low zoom level": 'Các địa điểm quan tâm sẽ bắt đầu xuất hiện ở mức thu phóng ${next} trở lên. Bạn đang xem mức thu phóng ${actual}.',
+                "all n entries": 'Tổng số {{total}} mục:',
+                "the first entries": '{{number}} mục đầu tiên trên tổng số {{total}} mục:',
+                "load all with JOSM": 'tải toàn bộ trong JOSM',
+                "evaluation tool": 'công cụ đánh giá',
+                "rule separator ;": 'ký tự phân tách quy tắc (quy tắc sau sẽ là quy tắc bình thường)',
+                "rule separator  ||": 'ký tự phân tách quy tắc (quy tắc sau sẽ là quy tắc dự trữ ứng với các thời gian bên ngoài các quy tắc trước)',
+                "rule separator ,": 'ký tự phân tách quy tắc (quy tắc sau sẽ là quy tắc bổ sung để kéo dài các quy tắc trước mà không ghi đè các quy tắc trước, khác với quy tắc bình thường)',
+                "JOSM remote conn error": 'Không thể kết nối với JOSM. Xin vui lòng kiểm tra rằng JOSM đang chạy và được thiết lập để điều khiển từ xa qua cổng mặc định 8111 của TCP.',
+                "refer to yohours": 'YoHours cũng có thể phân tích giá trị này, đây là một trình soạn thảo cú pháp opening_hours đơn giản. Nếu bạn không cần các cú pháp nâng cao, <a href=\"{{href}}\">hãy thử YoHours</a>.',
+                "include timestamp?": 'bao gồm dấu thời gian?',
+            },
+            "words": {
+                "modifier": "bộ thay đổi {{name}}",
+                "selector": "bộ chọn {{name}}",
+                "mode": "chế độ đánh giá",
+                "green": "xanh",
+                "yellow": "vàng",
+                "red": "đỏ",
+                "violet": "tím",
+                "to": "đến",
+                "and": "và",
+                "no": "không",
+                "undefined": "không định rõ",
+                // "his": undefined, // does not translate
```

Comment:
I have no idea what this string is used for; it doesn’t appear anywhere in this repository as far as I can tell.

Reply:
Git history for the rescue: `git log -p -S 'words.his'` -> It was used. If you want to, you can remove all "his" translations.

Reply:
I removed all the translations of this string.